moveit_ros/moveit_servo/config/ur_simulated_config.yaml

```diff
@@ -43,10 +43,10 @@ hard_stop_singularity_threshold: 30 # Stop when the condition number hits this
 joint_limit_margin: 0.1 # added as a buffer to joint limits [radians]. If moving quickly, make this larger.
 
 ## Topic names
-cartesian_command_in_topic: servo_server/delta_twist_cmds  # Topic for incoming Cartesian twist commands
-joint_command_in_topic: servo_server/delta_joint_cmds # Topic for incoming joint angle commands
-joint_topic:  joint_states
-status_topic: servo_server/status # Publish status to this topic
+cartesian_command_in_topic: delta_twist_cmds  # Topic for incoming Cartesian twist commands
+joint_command_in_topic: delta_joint_cmds # Topic for incoming joint angle commands
+joint_topic: joint_states
+status_topic: status # Publish status to this topic
 command_out_topic: joint_group_position_controller/command # Publish outgoing commands here
```

Comment:
Aren't the controller topics namespaced as well? A usual setup would have these at root level.

Reply:
Good point, I need to check the namespacing for this one.